modules/oxide/src/less/theme/content-ui.less

```diff
@@ -26,3 +26,4 @@
 @import 'content/table/table';
 @import 'content/visualblocks/visualblocks';
 @import 'content/visualchars/visualchars';
+@import 'content/list/list';
```

Comment:
It's been reverted now, but I'm be curious why we thought this change was necessary.

This was not an equivalent change. `content-ui.less` is imported by `skins/ui/*/content.less` not `skins/content/*/content.less` where the changes came from.

The whole point of keeping it in the content skins was so that customers could configure it instead of having it forced on them. Having it in the theme might make it seem more dynamic for us, but it's actually far more restrictive for customers.